### ISSUE TITLE
Command not found + input error

### ISSUE THREAD

**Issue by Dominoli** [mikehaertl/php-pdftk#58]
I'm experiencing 2 problems.

1. command not found (sh: pdftk)
Using the produced command (getCommand()) in console works tough.
Altering the command option, I'm getting "/usr/local/bin/pdftk: No such file or directory" with 'useExec => true' set.  Again, using that command in console does work...

2. input error
Commands are made like `pdftk 0='input1.pdf' 1='input2.pdf' output 'output.pdf'`, however, console does not seem to like 0=, 1= etc. producing an input error. Should command not be `pdftk 'input1.pdf' 'input2.pdf' output 'output.pdf`

I'm using version 2.02 of pdftk.

**Comment by mikehaertl** [comment]
1. You have to configure the path to `pdftk` in `command` if it can't be found from PHP. See https://github.com/mikehaertl/php-pdftk#execution
2. Please show the code you used to create this command.

**Comment by Dominoli** [comment]
Issue 1: solved. In Cloud Linux having all users caged with cageFS, you can only use commands of your home directory (or you can remove the user out of the cage which can pose a security thread). Copying pdftk to the /bin folder of the users home directory did the trick.

Issue 2: the '0=' and '1=' in front of the files are causing the problem. I also noticed that the output file in the command is not the one put in `$pdf->saveAs()`.
Example code:
```
use mikehaertl\pdftk\Pdf;
$output = "output.pdf";
$input = array("file1.pdf","file2.pdf");
$pdf = new Pdf($input,[
	    'command' => '/pathto/pdftk',
	    'useExec' => false
	]);
$pdf->saveAs($output);
```
**$pdf->getError():**
Error: Unable to find file.
Error: Failed to open PDF file: 
   0=file1.pdf
Error: Unable to find file.
Error: Failed to open PDF file: 
   1=file2.pdf
Errors encountered.  No output created.
Done.  Input errors, so no output created.

**$pdf->getCommand():**
```
mikehaertl\pdftk\Command Object
(
    [_command:protected] => /pathto/pdftk
    [_files:protected] => Array
        (
            [0] => Array
                (
                    [name] => file1.pdf
                    [password] => 
                )

            [1] => Array
                (
                    [name] => file2.pdf
                    [password] => 
                )

        )

    [_options:protected] => Array
        (
        )

    [_operation:protected] => 
    [_operationArgument:protected] => Array
        (
        )

    [_escapeOperationArgument:protected] => 
    [escapeArgs] => 1
    [escapeCommand] => 
    [useExec] => 
    [captureStdErr] => 1
    [procCwd] => 
    [procEnv] => 
    [procOptions] => 
    [locale] => 
    [_args:protected] => Array
        (
            [0] => 0='file1.pdf'
            [1] => 1='file2.pdf'
            [2] => output '/tmp/tmp_php_pdftk_mcjY3U.pdf'
        )

    [_execCommand:protected] => /pathto/pdftk 0='file1.pdf' 1='file2.pdf' output '/tmp/tmp_php_pdftk_mcjY3U.pdf'
    [_stdOut:protected] => 
    [_stdErr:protected] => Error: Unable to find file.
Error: Failed to open PDF file: 
   0=file1.pdf
Error: Unable to find file.
Error: Failed to open PDF file: 
   1=file2.pdf
Errors encountered.  No output created.
Done.  Input errors, so no output created.

    [_exitCode:protected] => 1
    [_error:protected] => Error: Unable to find file.
Error: Failed to open PDF file: 
   0=file1.pdf
Error: Unable to find file.
Error: Failed to open PDF file: 
   1=file2.pdf
Errors encountered.  No output created.
Done.  Input errors, so no output created.

    [_executed:protected] => 
)

```

**Comment by mikehaertl** [comment]
You need to alias your files, when you pass them to the constructor. See the second example here: https://github.com/mikehaertl/php-pdftk#cat

You can then reference your files later in the command, e.g. if you want to combine pages from several files.

**Comment by mikehaertl** [comment]
Hmm, I've now checked the man page of pdftk again. Here's the part where handles (what I called alias above) are mentioned:

```
<input PDF files | - | PROMPT>
       A list of the input PDF files. If you plan to combine these PDFs (without using handles) then list files in the order you want them combined.  Use -
       to pass a single PDF into pdftk via stdin.  Input files can be associated with handles, where a handle is one or more upper-case letters:

       <input PDF handle>=<input PDF filename>

       Handles are often omitted.  They are useful when specifying PDF passwords or page ranges, later.

       For example: A=input1.pdf QT=input2.pdf M=input3.pdf
```

The interesting part is "Input files **can** be associated with handles", so they should be optional. This is not yet supported here. 

If you can't use handles and need a fix, let me know if you could help testing. Fixing this is pretty simple.

**Comment by Dominoli** [comment]
> ..., where a handle is one or more upper-case letters

Made the coin drop :-)  I changed line 110 in class Pdf:
`if ($handle===null || is_numeric($handle)) {
    $handle = $this->nextHandle();
 }`

**Comment by mikehaertl** [comment]
Hmm, ok, should also work. I'd have changed the call to `addFile()` - but maybe your fix is even better. So you confirm that this fixes your issue? Then I'll adapt it to the library.

**Comment by Dominoli** [comment]
Yes, this fixed the problem and all working smooth now!

**Comment by mikehaertl** [comment]
Released 0.3.2 containing your fix. Thanks!
